javascript/javascript1/week1/myjavascript.js

```diff
@@ -0,0 +1,91 @@
+// Task 1 Age-ify
+const yearOfBirth = 1987;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//Task 2 Goodboy- Oldboy
+
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+const dogYear = dogYearFuture-dogYearOfBirth;
+const dogHumanYear = 7 * dogYear ;
+
+
+let shouldShowResultInDogYears = "true";
+
+if (shouldShowResultInDogYears == "true") {
+    console.log("Dog will be " + dogYear + " dog years old in " + dogYearFuture)
+} else {
+    console.log("My dog will be " + dogHumanYear+ " humuns years old in " + dogYearFuture)
+};
+
+//Task 3 Housey price
+
+//housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300; for reference
+
+// Peter house 
+
+const gardenSizeInM2 = 100 ;
+ const PWidth = 8;
+ const PHeight = 10;
+ const PDepth = 10;
+
+ //Calculation for Volume in Meters for Peter
+
+ let PvolumeInMeters = PWidth * PHeight * PDepth;
+
+ //Peter House Price
+
+ const actualPeterPrice = 2500000;
+
+ housePriceforPeter = PvolumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+ if (housePriceforPeter < actualPeterPrice){
+     console.log("Peter High price")
+ } else if (housePriceforPeter == actualPeterPrice){
```

Comment:
You can use "==="  strict equality operator in order to receive the error-free results.

Reply:
I have done the changes.

thanks